presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -99,6 +99,16 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    SQL statements support string value usage as simple literals ``Hello
+    world``. To use Unicode literals within a string, you have to use the prefix
+    ``U&`` for the completed string and prefix the Unicode character itself with
+    the escape character ``\``. For example, you can use ``U&`Glad to see you
```

Comment:
I dislike this because it states that the escape character is a backslash. We then have another paragraph that says it can be configured differently. I think it's better to call out backslash as a default, as I did in my original suggestion.